packages/backend/src/registries/ContainerRegistry.spec.ts

```diff
@@ -121,3 +136,27 @@ test('ContainerRegistry subscriber disposed should not be called', () => {
   // never should have been called
   expect(subscribeMock).toHaveBeenCalledTimes(0);
 });
+
+test('ContainerRegistry should fire ContainerStart when container start', () => {
+  // Get the callback created by the ContainerRegistry
+  let callback: (event: ContainerJSONEvent) => void;
+  mocks.onEventMock.mockImplementation((method: (event: ContainerJSONEvent) => void) => {
+    callback = method;
+  });
+
+  // Create the ContainerRegistry and init
+  const registry = new ContainerRegistry();
+  registry.init();
+
+  const startListenerMock = vi.fn();
+  registry.onStartContainerEvent(startListenerMock);
+
+  // Generate a remove event
+  callback({ status: 'remove', id: 'random', type: 'container' });
+
```

Comment:
I would add a `not.toHaveBeenCalled()` here, to be sure that startListenerMock has been called during the second call to `callback`, not the first one

Reply:
Yes good suggestion ! Thanks and added ✅  !